extensions/amp-story/1.0/amp-story.js

```diff
@@ -216,7 +216,9 @@ export class AmpStory extends AMP.BaseElement {
         new NavigationState(this.win, () => this.hasBookend_());
 
     /** @private {!AmpStoryAnalytics} */
-    this.analytics_ = new AmpStoryAnalytics(this.win, this.element);
+    this.analyticsService_ = new StoryAnalyticsService(this.win, this.element);
```

Comment:
This won't be a problem until other components require this service synchronously, but this reminds me of race conditions we had with the store service.
If another component require this service synchronously, and gets instantiated before `amp-story`, then it could fail. And because it's analytics, it could be a very long time before we realize it fails for certain documents. :(

Could we use something similar to what we do in the `amp-story-store-service`? https://github.com/ampproject/amphtml/blob/master/extensions/amp-story/1.0/amp-story-store-service.js#L29-L45